nirodents/data/artsBrainExtraction_precise_T2w.json

```diff
@@ -0,0 +1,61 @@
+{
+  "collapse_output_transforms": true,
+  "convergence_threshold": [1E-8, 1E-8, 1E-9],
+  "convergence_window_size": [10, 10, 15],
+  "dimension": 3,
+  "interpolation": "LanczosWindowedSinc",
+  "metric": [
+    "MI",
+    "MI",
+    ["CC", "CC"]
+  ],
+  "metric_weight": [
+    1,
+    1,
+    [0.5, 0.5]
+  ],
+  "number_of_iterations": [
+    [1000, 500, 250, 100],
+    [1000, 500, 250, 100],
+    [50, 10, 0]
+  ],
+  "output_transform_prefix": "anat_to_template",
+  "output_warped_image": true,
+  "radius_or_number_of_bins": [
+    32,
+    32,
+    [4, 4]
+  ],
+  "sampling_percentage": [
+    0.25,
+    0.25,
+    [1, 1]
+  ],
+  "sampling_strategy": [
+    "Regular",
+    "Regular",
+    ["None", "None"]
+  ],
+  "shrink_factors": [
+    [8, 4, 2, 1],
+    [8, 4, 2, 1],
+    [4, 2, 1]
+  ],
+  "sigma_units": ["vox", "vox", "vox"],
+  "smoothing_sigmas": [
+    [4, 2, 1, 0],
+    [4, 2, 1, 0],
+    [2, 1, 0]
+  ],
+  "transform_parameters": [
+    [0.1],
+    [0.1],
+    [0.1, 3.0, 0.0]
+  ],
+  "transforms": ["Rigid", "Affine", "SyN"],
+  "use_histogram_matching": true,
+  "verbose": true,
+  "winsorize_lower_quantile": 0.01,
```

Comment:
Same as the original, except for the lower quantile. Rodent images have a lot of power in this range. We should not discard as much.

Reply:
When we do antsRegistration we usually use 0.005 and 0.995, if that's of any use.